Px.Utils.UnitTests/PxFileTests/DataTests/StreamUtilitiesTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿using PxFileTests.Fixtures;
+using PxUtils.PxFile;
+using PxUtils.PxFile.Data;
+using System.Text;
+
+namespace PxFileTests.DataTests
+{
+    [TestClass]
+    public class StreamUtilitiesTests
+    {
+        /*
+         * THIS TEST SET ASSUMES THAT THE INPUT IS VALIDATED AND DOES NOT CONTAIN ANY ERRORS
+         */
+
+        [TestMethod]
+        public void FindKeywordTest_DataKeywordAtStartOfStream_ReturnsZero()
+        {
+            // Arrange
+            byte[] data = Encoding.UTF8.GetBytes("DATA=");
+            using Stream stream = new MemoryStream(data);
+
+            // Act
+            long position = StreamUtilities.FindKeywordPosition(stream, "DATA", PxFileSyntaxConf.Default);
+
+            // Assert
+            Assert.AreEqual(0, position);
+        }
+
+        [TestMethod]
+        public void FindKeywordTest_TwoDataKeywords_ReturnsZero()
+        {
+            // Arrange
+            byte[] data = Encoding.UTF8.GetBytes("DATADATA=");
+            using Stream stream = new MemoryStream(data);
+
+            // Act
+            long position = StreamUtilities.FindKeywordPosition(stream, "DATA", PxFileSyntaxConf.Default);
+
+            // Assert
+            Assert.AreEqual(-1, position);
+        }
+
+        [TestMethod]
+        public void FindKeywordTest_DataKeywordInTheMiddleOfStream_ReturnsIndex()
+        {
+            // Arrange
+            byte[] data = Encoding.UTF8.GetBytes("DATA=123");
```

Comment:
DATA keyword is not in the middle of the stream in this test case

Reply:
Fixed the test to match the description.